src/client/unittests/common/testConfigurationManager.ts

```diff
@@ -46,12 +58,12 @@ export abstract class TestConfigurationManager {
             matchOnDetail: true,
             placeHolder: 'Select the pattern to identify test files'
         };
-        let items: vscode.QuickPickItem[] = [
-            { label: '*test.py', description: `Python Files ending with 'test'` },
-            { label: '*_test.py', description: `Python Files ending with '_test'` },
-            { label: 'test*.py', description: `Python Files begining with 'test'` },
-            { label: 'test_*.py', description: `Python Files begining with 'test_'` },
-            { label: '*test*.py', description: `Python Files containing the word 'test'` }
+        const items: vscode.QuickPickItem[] = [
+            { label: '*test.py', description: 'Python Files ending with \'test\'' },
```

Comment:
Could use you double quotes instead of single quotes to avoid the quote escaping?

Reply:
Single quote is preferred in typescript (linters prefer escaping).
There are other options, but linter and best practice is (generally) to use single quotes for strings in js and ts world.

Reply:
Huh, interesting.